### PR TITLE
Feature/comment field

### PR DESCRIPTION
**Pull request by Shylock-Hg**
ONCALL-46

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]
Good job!

**Review by liuyu85cn — APPROVED** [review]
LGTM

File: src/common/clients/meta/MetaClient.h [hunk 1]
```diff
@@ -266,6 +266,7 @@ class MetaClient {
                    std::string indexName,
                    std::string tagName,
                    std::vector<cpp2::IndexFieldDef> fields,
+                   const std::string *comment = nullptr,
```
Comment (yixinglu) [inline src/common/clients/meta/MetaClient.h:269]: Prefer to put the parameter behind of `ifNotExists` for interface compatibility

Comment (Shylock-Hg) [reply src/common/clients/meta/MetaClient.h]: Solved.

**Review by darionyaphet — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**